api/src/paths/project/{projectId}/survey/create.ts

```diff
@@ -100,6 +102,10 @@ POST.apiDoc = {
             ecological_season_id: {
               type: 'number'
             },
+            surveyed_all_areas: {
+              type: 'string',
+              enum: ['true', 'false']
```

Comment:
Any particular reason why we're encoding `surveyed_all_areas` as an enum instead of a boolean?

Reply:
Its mainly because the MUI radio controls don't support boolean values, as dumb as that is.

So at some point in the process we have to convert from string to boolean when saving, and vice versa when fetching. I just landed on doing it in the API, mainly because converting on the frontend seemed like more effort and we don't have a pattern in place currently to handle these kinds of conversions on the frontend (like a model to translate fields, etc). Something we could look into though.